src/components/productCategoriesComponent.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import Carousel from '../components/ui/cardCarouselComponent';
+
+const ProductCategories = (props) => {
```

Comment:
destructure props here 
```suggestion
const ProductCategories = ( {featured} ) => {
```